tests/integration/testlib/assertlib.py

```diff
@@ -53,3 +54,17 @@ def assert_absent(*ifnames):
 
     current_state = statelib.show_only(ifnames)
     assert not current_state[INTERFACES]
+
+
+def iface_macs(state):
```

Comment:
nit: I like when the caller is before the callee, could you please place this below the assert function?

It should also be a private.

Reply:
private?

Reply:
`_iface_macs`

Reply:
That's what I did. I mean I though you wanted to use the generator function in test files

Reply:
What do you mean this is what you have done? Private means to prefix with `_`.

Reply:
I meant that I had a prefix `_` before but I though that you didn't want to make the generator function private. It's all good now